app/constants/index.ts

```diff
@@ -41,6 +42,10 @@ export const MAX_STACKING_CYCLES = 12;
 
 export const MIN_STACKING_CYCLES = 1;
 
+export const MIN_DELEGATED_STACKING_AMOUNT_USTX = 50_000_000;
+
+export const MAX_DELEGATED_STACKING_AMOUNT_USTX = stxToMicroStx(10_000_000_000);
```

Comment:
Why even have this at all? If someone has 10m STX, they should be able to do whatever they want with it IMO

Reply:
it's 50 stx, right?

Reply:
I'm thinking having no upper limit might open up some big number issues. Can you stack Infinity STX? This is 10bn stx.